packages/docs/src/css/custom.css

```diff
@@ -37,3 +37,7 @@
 [data-theme='dark'] .docusaurus-highlight-code-line {
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+p {
+  margin: 0;
```

Comment:
Have you checked all pages to see if there are no weird rendering?

Reply:
Yes, looks good